HST Office packaging: LibreLogo packaging with --enable-python=system.

ENABLE_LIBRELOGO is only passed to the scp2 preprocessor in the bundled-Python
branch of InstallModule_python.mk. With a system Python, gid_Module_Pyuno
therefore picks packinfo_librelogo_disabled.txt (the pyuno .deb Conflicts/
Replaces <prefix>-librelogo) while the librelogo module is still packaged,
so pyuno and librelogo cannot be installed together and the .deb set is
uninstallable. Pass the define in the system-Python branch too.

License: modified upstream file keeps its license (MPL-2.0).
---
--- a/scp2/InstallModule_python.mk
+++ b/scp2/InstallModule_python.mk
@@ -22,6 +22,7 @@
 else ifneq ($(SYSTEM_PYTHON),)
 $(eval $(call gb_InstallModule_add_defs,scp2/python,\
 	-DSYSTEM_PYTHON \
+	$(call gb_Helper_optional,LIBRELOGO,-DENABLE_LIBRELOGO=1) \
 ))
 
 else
--- a/scp2/source/calc/folderitem_calc.scp
+++ b/scp2/source/calc/folderitem_calc.scp
@@ -20,7 +20,7 @@
 
 #ifndef ENABLE_SILENT_MSI
 FolderItem gid_Folderitem_Scalc
-    Name = "%PRODUCTNAME Calc";
+    Name = "HST Sheets";
     ModuleID = gid_Module_Prg_Calc_Bin;
     FolderID = gid_Folder_Staroffice51;
     FileID = auto_calc_brand_exe_scalc;
--- a/scp2/source/calc/module_calc.ulf
+++ b/scp2/source/calc/module_calc.ulf
@@ -16,26 +16,26 @@
  *   the License at http://www.apache.org/licenses/LICENSE-2.0 .
  */
 [STR_NAME_MODULE_PRG_CALC]
-en-US = "%PRODUCTNAME Calc"
+en-US = "%PRODUCTNAME Sheets"
 
 [STR_DESC_MODULE_PRG_CALC]
-en-US = "Perform calculations, analyze information and manage lists in spreadsheets by using %PRODUCTNAME Calc."
+en-US = "Perform calculations, analyze information and manage lists in spreadsheets by using %PRODUCTNAME Sheets."
 
 [STR_NAME_MODULE_PRG_CALC_BIN]
 en-US = "Program Module"
 
 [STR_DESC_MODULE_PRG_CALC_BIN]
-en-US = "The application %PRODUCTNAME Calc"
+en-US = "The application %PRODUCTNAME Sheets"
 
 [STR_NAME_MODULE_PRG_CALC_HELP]
-en-US = "%PRODUCTNAME Calc Help"
+en-US = "%PRODUCTNAME Sheets Help"
 
 [STR_DESC_MODULE_PRG_CALC_HELP]
-en-US = "Help about %PRODUCTNAME Calc"
+en-US = "Help about %PRODUCTNAME Sheets"
 
 [STR_NAME_MODULE_OPTIONAL_ADDINS]
 en-US = "Add-in"
 
 [STR_DESC_MODULE_OPTIONAL_ADDINS]
-en-US = "Add-ins are additional programs that make new functions available in %PRODUCTNAME Calc."
+en-US = "Add-ins are additional programs that make new functions available in %PRODUCTNAME Sheets."
 
--- a/scp2/source/impress/folderitem_impress.scp
+++ b/scp2/source/impress/folderitem_impress.scp
@@ -20,7 +20,7 @@
 
 #ifndef ENABLE_SILENT_MSI
 FolderItem gid_Folderitem_Simpress
-    Name = "%PRODUCTNAME Impress";
+    Name = "HST Slides";
     ModuleID = gid_Module_Prg_Impress_Bin;
     FolderID = gid_Folder_Staroffice51;
     FileID = auto_impress_brand_exe_simpress;
--- a/scp2/source/impress/module_impress.ulf
+++ b/scp2/source/impress/module_impress.ulf
@@ -18,22 +18,22 @@
 
 
 [STR_NAME_MODULE_PRG_IMPRESS]
-en-US = "%PRODUCTNAME Impress"
+en-US = "%PRODUCTNAME Slides"
 
 [STR_DESC_MODULE_PRG_IMPRESS]
-en-US = "Create and edit presentations for slideshows, meeting and Web pages by using %PRODUCTNAME Impress."
+en-US = "Create and edit presentations for slideshows, meeting and Web pages by using %PRODUCTNAME Slides."
 
 [STR_NAME_MODULE_PRG_IMPRESS_BIN]
 en-US = "Program Module"
 
 [STR_DESC_MODULE_PRG_IMPRESS_BIN]
-en-US = "The application %PRODUCTNAME Impress"
+en-US = "The application %PRODUCTNAME Slides"
 
 [STR_NAME_MODULE_PRG_IMPRESS_HELP]
-en-US = "%PRODUCTNAME Impress Help"
+en-US = "%PRODUCTNAME Slides Help"
 
 [STR_DESC_MODULE_PRG_IMPRESS_HELP]
-en-US = "Help about %PRODUCTNAME Impress"
+en-US = "Help about %PRODUCTNAME Slides"
 
 
 
--- a/scp2/source/math/folderitem_math.scp
+++ b/scp2/source/math/folderitem_math.scp
@@ -20,7 +20,7 @@
 
 #ifndef ENABLE_SILENT_MSI
 FolderItem gid_Folderitem_Smath
-    Name = "%PRODUCTNAME Math";
+    Name = "HST Math";
     ModuleID = gid_Module_Prg_Math_Bin;
     FolderID = gid_Folder_Staroffice51;
     FileID = auto_math_brand_exe_smath;
--- a/scp2/source/writer/folderitem_writer.scp
+++ b/scp2/source/writer/folderitem_writer.scp
@@ -20,7 +20,7 @@
 
 #ifndef ENABLE_SILENT_MSI
 FolderItem gid_Folderitem_Swriter
-    Name = "%PRODUCTNAME Writer";
+    Name = "HST Writer";
     ModuleID = gid_Module_Prg_Wrt_Bin;
     FolderID = gid_Folder_Staroffice51;
     FileID = auto_writer_brand_exe_swriter;
--- a/scp2/source/draw/folderitem_draw.scp
+++ b/scp2/source/draw/folderitem_draw.scp
@@ -20,7 +20,7 @@
 
 #ifndef ENABLE_SILENT_MSI
 FolderItem gid_Folderitem_Sdraw
-    Name = "%PRODUCTNAME Draw";
+    Name = "HST Draw";
     ModuleID = gid_Module_Prg_Draw_Bin;
     FolderID = gid_Folder_Staroffice51;
     FileID = auto_draw_brand_exe_sdraw;
--- a/scp2/source/base/folderitem_base.scp
+++ b/scp2/source/base/folderitem_base.scp
@@ -20,7 +20,7 @@
 
 #ifndef ENABLE_SILENT_MSI
 FolderItem gid_Folderitem_Sbase
-    Name = "%PRODUCTNAME Base";
+    Name = "HST Base";
     ModuleID = gid_Module_Prg_Base_Bin;
     FolderID = gid_Folder_Staroffice51;
     FileID = auto_base_brand_exe_sbase;
